test/integration/freetext_editor_spec.mjs

```diff
@@ -56,9 +56,6 @@ const copyPaste = async page => {
   await kbCopy(page);
   await promise;
 
-  // eslint-disable-next-line no-restricted-syntax
-  await waitForTimeout(10);
```

Comment:
Note that for both copy and paste we already wait for the corresponding event (above and below this block), so this occurrence can therefore be removed without replacement.